### ISSUE TITLE
Investigate unknown issue for checkv/updatedatabase module

### ISSUE THREAD

**Issue by maxulysse** [nf-core/modules#5972]
Unknown issue on `docker_self_hosted` for:

- `checkv/updatedatabase`

_Originally posted by @maxulysse in https://github.com/nf-core/modules/issues/5939#issuecomment-2225318339_

**Comment by maxulysse** [comment]
@jfy133 that's one of the modules you use, right?

**Comment by Joon-Klaps** [comment]
Yeah, I use `checkv end_to_end` and `checkv downloaddatabase`. However, I've not seen this issue with `checkv updatedatabase` ... . I can't reproduce it locally and on github actions, I'm a bit lost to debug as it mentions that it was unsuccessful but I can't access the raw log files. 

```
  >  -- Check '/home/runner/work/modules/modules/.nf-test/tests/c9b6949340ad2c06d8182afaf02075d8/meta/nextflow.log' file for details
Assertion failed: 

assert process.success
       |       |
       |       false
       CHECKV_UPDATEDATABASE

java.lang.ArrayIndexOutOfBoundsException: Array index out of range: 0
java.lang.ArrayIndexOutOfBoundsException: Array index out of range: 0
java.lang.ArrayIndexOutOfBoundsException: Array index out of range: 0
    FAILED (35.087s)
``` 

So yes, this is still an issue, and I don't know how to fix it without Github actions' [artifacts](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow).

**Comment by vagkaratzas** [comment]
Still an issue with this one?

**Comment by Joon-Klaps** [comment]
@vagkaratzas idk, think it can be closed? 
The module got a version updated a month or 2 after I replied in this thread
https://github.com/nf-core/modules/pull/8648

**Comment by vagkaratzas** [comment]
Let's close for now, and re-open if in the future if necessary.
